### PR TITLE
More fixes for Windows

### PR DESCRIPTION
**Pull request by alex85k**
After merging different PRs some more incompatibilities with MS Visual Studio arises.
Here they are, with comments.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -93,6 +90,7 @@ else()
     ADD_DEFINITIONS("-std=c++11")
     #ADD_DEFINITIONS("-fdiagnostics-color=auto")
     ADD_DEFINITIONS(-DBOOST_TEST_DYN_LINK)
+    ADD_DEFINITIONS(-DBOOST_LOG_DYN_LINK)
```
Comment (alex85k) [inline CMakeLists.txt]: Let people choose dynamic/static Boost, their home-made packages may contain only .lib or .dll-s too.

File: include/server/http_server.hpp [hunk 2]
```diff
@@ -24,6 +24,7 @@
 
 #include "settings.hpp"
 
+#include <set>
```
Comment (alex85k) [inline include/server/http_server.hpp]: had compile error "unknown std::set"

File: src/server/renderer/render_canvas.cpp [hunk 3]
```diff
@@ -31,7 +31,7 @@
 
 #include "config.hpp"
 
-#include <cairo/cairo-svg.h>
+#include <cairo-svg.h>
```
Comment (alex85k) [inline src/server/renderer/render_canvas.cpp]: Our cmake seems to find path directly to cairo includes. Let's see what Travis CI says.

**Comment by florianjacob** [comment]
Looks good, thanks.

**Pull request merged**